src/main/resources/assets/creeperspores/lang/fr_fr.json

```diff
@@ -0,0 +1,5 @@
+{
+  "effect.creeperspores.creeper_spore": "Spores de creeper",
+  "entity.creeperspores.creeperling": "Creeperling",
```

Comment:
Ah but you can't just nope out of this !
How about "Creepetit" ?

Reply:
I didn't find any good translation for the -ling suffix, so I thought it could stay like that, especially since Creeper is perfectly translatable but the base game lets it stay in english.

Will update then.